### ISSUE TITLE
Import / Export phonebook - Multi-language and Excel

### ISSUE THREAD

**Issue by starnetwork** [playsms/playsms#304]
Hi,
the Standard CSV Import / Export is not working with Hebrew (in my opinion also in other none English languages)
when we opening the csv file in Excel, He is known as a very popular way to open csv / tables
Please check http://forums.sugarcrm.com/f6/export-record-csv-excel-file-utf-8-a-81858/
and consider to add one of the following options:
1. another Import / Export support for XLS files, using for example http://phpexcel.codeplex.com/
2. add option to configure the encoding for csv, for Example Windows-1255 is good for Hebrew.

All the best,
Star Network.

**Comment by yosef454** [comment]
Another problem Send from file.
If a file to an in Hebrew it does not send the message.
I converted file to UTF8 with notepad ++, it sent a blank message.

**Comment by yosef454** [comment]
I found it,

Add UTF-8 CSV import support.
https://github.com/yahia007/playSMS/commit/499a1d75674155d5174093d7f4bdc64b64ddd527

Add support for exporting UTF-8 CSV
https://github.com/yahia007/playSMS/commit/92e0ef71497b9b384f376abaad55c67a8f82191b

**Comment by yosef454** [comment]
Hi anton,

This change does not solve the problem.
You might need to set encoding UTF-16LE", 'UTF-8?

**Comment by antonraharja** [comment]
yes, it does not. I wasn't trying to fix it, I just make changes based on your change to core_download()

what you did was very specific to your need, while core_download() used for generic purposes, thats why I made everything configurable. it is the user of core_download() that need to be adjusted, in this case on import.php

anton
